bgt-loader/src/main/java/nl/b3p/brmo/util/http/HttpClientWrappers.java

```diff
@@ -28,8 +28,8 @@ public static HttpClientWrapper getDefault() {
         if ("urlconnection".equals(System.getProperty("httpclientwrapper"))) {
             return new URLConnectionHttpClientWrapper();
         }
-        Float f = Float.parseFloat(System.getProperty(("java.specification.version")));
-        if (f != null && f >= 11.0f) {
+        float f = Float.parseFloat(System.getProperty(("java.specification.version")));
+        if (f >= 11.0f) {
```

Comment:
deze check is niet echt nodig; baseline voor BRMO is Java 11 (sinds #1090), zie ook 
https://github.com/B3Partners/brmo/blob/b6c99e5498341227bf7374048154052ad10d1d94/pom.xml#L57-L59

Reply:
Dat klopt, maar ik wil deze code afsplitsen in een apart project.